src/main/java/core/basesyntax/ConsoleHandler.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    private Dao dao;
+    private String command;
+    private Scanner scanner;
+    private final Splitter splitter;
```

Comment:
let's make `command` and `scanner` local variables
they have no connection to ConsoleHandler instance

also, why is splitter final?